app/src/main/java/ch/epfl/sweng/erpa/services/firebase/FirebaseRemoteServicesProvider.java

```diff
@@ -15,4 +17,14 @@ public String getFriendlyProviderName() {
     public String getFriendlyProviderDescription() {
         return "Your phone will connect directly to Firebase, Google Play Services is required for this to work";
     }
+
+    @Override
+    public Optional<String> getUidFromUsername(String username) {
+        return null;
```

Comment:
Since it's an optional, can you please return an `Optional.Empty`, does it make sense?